dolphinscheduler-master/src/main/java/org/apache/dolphinscheduler/server/master/registry/ServerNodeManager.java

```diff
@@ -227,6 +221,16 @@ public void run() {
         }
     }
 
+
+    protected void handleAddr(Map<String, String> newWorkerNodeInfo, WorkerGroup wg, Set<String> nodes) {
+        String[] addrs = wg.getAddrList().split(Constants.COMMA);
+        for (String addr : addrs) {
+            if (newWorkerNodeInfo.containsKey(addr)) {
+                nodes.add(addr);
+            }
+        }
+    }
```

Comment:
it's better to rename this method to `protected Set<String> getNodeAddress(Map<String, String> newWorkerNodeInfo, WorkerGroup workgroup)`